src/server/translate.ts

```diff
@@ -10,8 +10,11 @@ import fetch from 'node-fetch';
 import queryString from 'query-string';
 import FormData from 'form-data';
 import { ApiTranslateType } from '../interfaces';
+import config from '../config';
 
-const baseUrl = 'https://ndla.norskrobot.no:3443';
+const baseUrl = config.ndkmUser
+  ? 'https://ndla.norskrobot.no:4443'
+  : 'https://ndla.norskrobot.no:3443';
```

Comment:
Sikkert greit å spesifisere hvorfor vi må bruke forskjellige porter også

Reply:
Tja. Er jo fordi det er to varianter av tjensten som bruker forskjellig autentisering. Dette er uansett et mellomstadie. 3443 forsvinner når denne er i prod.